lib/devise/async.rb

```diff
@@ -31,6 +31,10 @@ module Backend
     mattr_accessor :enabled
     @@enabled = true
 
+    # Defines the delayed job priority.
+    mattr_accessor :delayed_job_priority
+    @@delayed_job_priority = 0
```

Comment:
Is it maybe reasonable to use the default priority that is set by Delayed Job itself?
```ruby
Delayed::Worker.default_priority
```